composer.json

```diff
@@ -23,8 +23,9 @@
         }
     ],
     "require": {
-        "mtdowling/cron-expression": "~1.0",
-        "silverstripe/framework": "^4"
+        "dragonmantank/cron-expression": "^3",
+        "silverstripe/framework": "^4",
+        "php": "^7.1 || 8"
```

Comment:
have we confirmed that everything works on php8?

Reply:
Not yet, but I'm not sure that's a good reason to block installation on PHP 8 systems.

Reply:
...but adding PHP 8 to the build matrix is probably the best idea ;)

Reply:
OK I've added it and it looks like https://github.com/silverstripe/silverstripe-framework/pull/9665 will need to be merged first, which is unsurprising.

Suggest we hold of merging this until then.

Reply:
yeah it's a bit awkward including php8 because it being there implies to end-users that it's compatible, though at the same time we want it to be there there so that we can install it and test in, particularly in places like travis

Reply:
Yeah the linked PR shouldn't be too far off, so I think it's fine to wait for that.